app/uk/gov/hmrc/brm/config/BrmConfig.scala

```diff
@@ -48,18 +48,9 @@ trait BrmConfig extends ServicesConfig {
   def ignoreMiddleNames : Boolean = getConfBool("birth-registration-matching.matching.ignoreMiddleNames",
```

Comment:
@manishsw Can we please rename this switch to ignoreAdditionalNames as per the interface.